lib/pack/webpack/webpack.go

```diff
@@ -301,7 +301,7 @@ func (s *Server) needsAuth(fn authHandle) httprouter.Handle {
 		if !hasCookie {
 			user, checker, err = s.cfg.Users.AuthenticateUser(*authCreds)
 			if err != nil {
-				log.Debugf("authenticate error: %v", err)
+				log.Debugf("authenticate error: %v", trace.DebugReport(err))
```

Comment:
nit: `log.WithError(err)` will keep the stack formatting - otherwise the it will be escaped and barely readable.

Reply:
`| sed s/\\n/\n/g' | sed 's/\\t/\t/g' my friend :). I think for this PR, I don't want to go through the entire file and clean up the errors logged, I think that should be it's own PR.